docs/asdf/extending/extensions.rst

```diff
@@ -222,8 +222,6 @@ an additional list of class names that previously identified the extension:
 Making converted object's contents visible to ``info`` and ``search``
 ---------------------------------------------------------------------
 
-When an object is converted to YAML, the resulting YAML tree is stored in the
```

Comment:
I noticed this stray sentence fragment and figured I'd stuff that fix into this PR.